plantower/plantower.py

```diff
@@ -18,6 +18,10 @@
 MSG_CHAR_1 = b'\x42' # First character to be recieved in a valid packet
 MSG_CHAR_2 = b'\x4d' # Second character to be recieved in a valid packet
 
+PMS_PASSIVE_MODE = 0
+PMS_ACTIVE_MODE = 1
+
+
```

Comment:
I would suggest to add the commands as global variables for consistency

```PMS_CMD_CHANGEMODE_PASSIVE = b'\x42\x4d\xe1\x00\x00\x01\x70'
PMS_CMD_CHANGEMODE_ACTIVE = b'\x42\x4d\xe1\x00\x01\x01\x71'
PMS_CMD_READ_IN_PASSIVE = b'\x42\x4d\xe2\x00\x00\x01\x71'
PMS_CMD_TO_SLEEP = b'\x42\x4d\xe4\x00\x00\x01\x73'
PMS_CMD_TO_WAKEUP = b'\x42\x4d\xe4\x00\x01\x01\x74'
```

Reply:
ok